yarn.lock

```diff
@@ -6285,18 +6285,6 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@microsoft/tsdoc-config@npm:~0.15.2":
```

Comment:
can you sync with main?  or exclude this file?

Reply:
yes - updated.